gherkin-utils/java/gherkin-utils/src/main/java/gherkin/utils/walker/GherkinDocumentWalker.java

```diff
@@ -0,0 +1,397 @@
+package gherkin.utils.walker;
+
+import gherkin.utils.walker.model.DefaultFilters;
+import gherkin.utils.walker.model.DefaultHandlers;
+import io.cucumber.messages.Messages.GherkinDocument;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+
+public class GherkinDocumentWalker {
+    private final DefaultFilters filters;
+    private final DefaultHandlers handlers;
+
+    public GherkinDocumentWalker(DefaultFilters iFilters, DefaultHandlers iHandlers) {
+        this.filters = iFilters;
```

Comment:
No need to add the `i` prefix.

Reply:
Sr, this is my mistake. I have already fixed these parameter names.